miniapps/nurbs/nurbs_ex1.cpp

```diff
@@ -169,6 +170,8 @@ int main(int argc, char *argv[])
                   "Master boundaries for periodic BCs");
    args.AddOption(&slave, "-ps", "--slave",
                   "Slave boundaries for periodic BCs");
+   args.AddOption(&neu, "-n", "--neu",
```

Comment:
Should there be a sample run with this new feature?

Reply:
Good idea will do that.

Reply:
Test case `nurbs_ex1_neu_r2_ser` has been added to check the functionality.
This run has also been added as an example. 
`nurbs_ex1 -m ../../data/square-disc-nurbs.mesh -o -1`